modules/ROOT/pages/java-support.adoc

```diff
@@ -33,6 +33,25 @@ The maximum supported version for running your applications is Java 17 for Mule
 
 For more information about Mule release cadences, see xref:release-notes::mule-runtime/lts-edge-release-cadence.adoc[].
 
+== Steps for Upgrade 
+
+Follow these steps to upgrade to Java 17:
+
+. Learn about <<application-performance, application performance>> when upgrading to Java 17. 
```

Comment:
Why is this top of the list?

Reply:
why are the performance consideration something the users need to consider before starting? There are a lot of other activities to consider before that

Reply:
Can you please list some?

Reply:
Pretty much everything else. Customers can't really evaluate the performance considerations until they actually understand the steps to take and their implications, specially when the upgrade requires changes to their applications.

Reply:
I'll move the step